### ISSUE TITLE
Cannot run `rake test` with JRuby 9.2

### ISSUE THREAD

**Issue by azumakuniyuki** [sisimai/rb-sisimai#270]
```
% /usr/local/jruby-9.2/bin/jruby -v
jruby 9.2.21.0 (2.5.8) 2022-06-27 49e5080a7c Java HotSpot(TM) 64-Bit Server VM 11.0.5+10-LTS on 11.0.5+10-LTS +jit [darwin-x86_64]
```

```
% PATH="$PATH:/usr/local/jr/bin" /usr/local/jr/bin/rake --trace test

rake aborted!
LoadError: no such file to load -- bundler/gem_helper
org/jruby/RubyKernel.java:974:in `require'
/usr/local/jruby-9.2/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:83:in `require'
/Users/ak/workingcopy/rr.nyaan.jp/rb-sisimai/Rakefile:1:in `<main>'
org/jruby/RubyKernel.java:1009:in `load'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/rake_module.rb:29:in `load_rakefile'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:734:in `raw_load_rakefile'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:126:in `block in load_rakefile'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:208:in `standard_exception_handling'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:125:in `load_rakefile'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:82:in `block in run'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:208:in `standard_exception_handling'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/lib/rake/application.rb:80:in `run'
/usr/local/jruby-9.2/lib/ruby/gems/shared/gems/rake-13.1.0/exe/rake:27:in `<main>'
org/jruby/RubyKernel.java:1009:in `load'
/usr/local/jr/bin/rake:23:in `<main>'
PATH="$PATH:/usr/local/jr/bin" /usr/local/jr/bin/rake --trace test  14.80s user 0.71s system 277% cpu 5.583 total
```

**Comment by azumakuniyuki** [comment]
`rake test` properly works on JRuby 9.3

```
% /usr/local/jr/bin/jruby -v
jruby 9.3.2.0 (2.6.8) 2021-12-01 0b8223f905 Java HotSpot(TM) 64-Bit Server VM 11.0.5+10-LTS on 11.0.5+10-LTS +jit [darwin-x86_64]

% PATH="$PATH:/usr/local/jr/bin" /usr/local/jr/bin/rake test
Run options: --seed 50957

# Running:

.. ***warning: Failed to strptime Thu, 28 May 2020 23:07:59
F. ***warning: Failed to strptime Wed, 16 Oct 2013 14:15:35
```

**Comment by hiroyuki-sato** [comment]
Exec `gem install bundler` before rake test or Update bundler

```
GEM_HOME=/tmp/gem java -jar ~/.embulk/jars/jruby-complete-9.2.20.1.jar -S gem install bundler -v 2.3.27
```

```
GEM_HOME=/tmp/gem java -jar ~/.embulk/jars/jruby-complete-9.2.20.1.jar -S rake test
```


```
GEM_HOME=/tmp/gem java -jar ~/.embulk/jars/jruby-complete-9.2.20.1.jar -S rake test
2024-03-06T14:56:58.385+09:00 [main] WARN FilenoUtil : Native subprocess control requires open access to the JDK IO subsystem
Pass '--add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED' to enable.
Run options: --seed 26554

# Running:

/private/tmp/rb-sisimai/lib/sisimai/lhost/exchange2007.rb:150: warning: `-' after local variable or literal is interpreted as binary operator
/private/tmp/rb-sisimai/lib/sisimai/lhost/exchange2007.rb:150: warning: even though it seems like unary operator
 ***warning: Failed to strptime Thu, 29 Apr 2013 23:45:51
F.........................................E.. ***warning: Failed to strptime Thu, 18 Sep 2008 17:54:04
 ***warning: Failed to strptime Tue, 28 Apr 2009 08:17:48
 ***warning: Failed to strptime Thu, 5 Mar 2009 06:28:13
 ***warning: Failed to strptime Wed, 17 Sep 2008 22:25:40
...

```

**Comment by azumakuniyuki** [comment]
This issue resolved at v5.0.2
